src/components/ResourceList/components/FilterControl/components/DateSelector/tests/DateSelector.test.tsx

```diff
@@ -179,6 +208,15 @@ describe('<DateSelector />', () => {
   });
 
   describe('onFilterValueChange', () => {
+    let getTimezoneOffset: jest.SpyInstance;
+    beforeEach(() => {
+      getTimezoneOffset = jest.spyOn(Date.prototype, 'getTimezoneOffset');
+    });
```

Comment:
You could move the mock one level up, this way you don't have to define it twice for two different `describe`